### PR TITLE
Create azure-ad-conditional-access-client-mobile-new.PNG

### PR DESCRIPTION
**Pull request by msbemba**
Client apps are no longer in preview and the options have changed

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3219

File: microsoft-365/solutions/create-secure-guest-sharing-environment.md [hunk 1]
```diff
@@ -160,8 +160,8 @@ To restrict guests to web-ony access
 9. On the **Cloud apps or actions** blade, click **Done**.
 10. Under **Assignments**, click **Conditions**.
 11. On the **Conditions** blade, click **Client apps**.
-12. On the **Client apps** blade, click **Yes** for **Configure**, and then select the **Mobile apps and desktop clients** and **Modern authentication clients** settings.</br>
-    ![Screenshot of Azure AD conditional access client apps settings](../media/azure-ad-conditional-access-client-mobile.png)
+12. On the **Client apps** blade, click **Yes** for **Configure**, and then select the **Mobile apps and desktop clients** settings.</br>
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/create-secure-guest-sharing-environment.md]:
```suggestion
12. On the **Client apps** blade, click **Yes** for **Configure**, and then select the **Mobile apps and desktop clients** setting.</br>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks.

**Comment by MikePlumleyMSFT** [comment]
@msbemba, file names all need to be lower case. Please update the .PNG file to .png. Also, no need to change the filename. Let's keep azure-ad-conditional-access-client-mobile.png. Thanks.

**Comment by msbemba** [comment]
@MikePlumleyMSFT I have renamed the file to .png and retained the same file name. Let me know if it looks good now.

**Comment by MikePlumleyMSFT** [comment]
@msbemba, looks good, thanks.

**Pull request merged**